make.bat

```diff
@@ -0,0 +1,35 @@
+@ECHO OFF

```

Comment:
Do we need this file?

Reply:
yes, for building in windows, and This is auto-generated from sphinx-init